buf/registry/module/v1beta1/label_service.proto

```diff
@@ -132,6 +135,8 @@ message ListLabelHistoryRequest {
   // If empty, the first page is returned.
   string page_token = 2 [(buf.validate.field).string.max_len = 4096];
   // The Label to list history for.
+  //
+  // This can reference archived Labels.
```

Comment:
I just noticed that we allow querying the history of an archived label. I think this is right/necessary, but also seems to contradict a comment above about `If a Label is archived, the API considers it to not exist, except when calling LabelService.GetLabels`.

I am wondering if we need to update that comment to say GetLabels is is the only endpoint that can _return_ an archived label.

Reply:
just to be consistent
```suggestion
  // This may reference archived Labels.
```

Reply:
I think that we make it clear that archived labels can only be retrieved by `LabelService.GetLabels` in our documentation. When an RPC takes 1+ label(s) as an input, it always accepts archived labels (e.g. `CreateOrUpdateLabel`, `ListLabelHistory`, etc.) so I think this is helpful clarifying documentation, but does not create any inconsistencies to the mental model.